### PR TITLE
Adds Dynamic Workers Ubuntu 18.04 end-of-life notification

### PR DESCRIPTION
**Pull request by goosmurf**
This PR adds notification of the end-of-life of Ubuntu 18.04 Dynamic Worker images and the timeline to transition to Ubuntu 22.04 images.

### CODE REVIEW

File: redirects.txt [hunk 1]
```diff
@@ -1115,4 +1115,5 @@ docs/projects/version-control/configuring-version-control-on-a-project.md -> doc
 docs/getting-started/first-deployment/configure-environments.md -> docs/getting-started/first-deployment/legacy-guide/configure-environments.md
 docs/getting-started/first-deployment/create-and-deploy-a-release.md -> docs/getting-started/first-deployment/legacy-guide/create-and-deploy-a-release.md
 docs/getting-started/first-deployment/create-projects.md -> docs/getting-started/first-deployment/legacy-guide/create-projects.md
-docs/getting-started/first-deployment/define-the-deployment-process.md -> docs/getting-started/first-deployment/legacy-guide/define-the-deployment-process.md
\ No newline at end of file
+docs/getting-started/first-deployment/define-the-deployment-process.md -> docs/getting-started/first-deployment/legacy-guide/define-the-deployment-process.md
+docs/notifications/dynamic-workers-ubuntu-1804-deprecation.md -> docs/infrastructure/workers/dynamic-worker-pools/ubuntu-1804-end-of-life.md
```
Comment (goosmurf) [inline redirects.txt:1119]: This is needed because in PreProd this page was initially on `docs/notifications/dynamic-workers-ubuntu-1804-deprecation.md`.

Comment (flin-8) [reply redirects.txt:1119]: oh so you're "committed" to that previous change :(

File: docs/infrastructure/workers/dynamic-worker-pools/ubuntu-1804-end-of-life.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+﻿---
+title: Ubuntu 18.04 End-of-life
+description: Describing the deprecation process of Ubuntu 18.04 Dynamic Workers.
+position: 50
+hideInThisSection: true
+hideInThisSectionHeader: true
+---
+
+Our Ubuntu Dynamic Workers are being upgraded to use Ubuntu 22.04, this upgrade will result in breaking changes for users of gcloud CLI and users of .NET Core 2.1/3.1 and Ubuntu 18.04 capabilities that are not offered by the updated replacements.
```
Comment (flin-8) [inline docs/infrastructure/workers/dynamic-worker-pools/ubuntu-1804-end-of-life.md:9]: It looks like we're using lower case `dynamic workers` everywhere - let's make it consistent then

**Review by flin-8 — APPROVED** [review]
LGTM + comment

**Pull request merged**